internal/compiler-interface/src/main/contraband/incremental.json

```diff
@@ -103,13 +103,6 @@
           ]
         },
         {
-          "name": "nameHashing",
```

Comment:
**Please** can we have a deprecation cycle when we remove things??

Upgrading sbt/sbt (and I assume every other client of sbt/zinc) is such a PITA if we change things like this.